### PR TITLE
Mockups

### PR DESCRIPTION
**Pull request by kanafghan**
The documentation of the mockups for the first version of the app.

### CODE REVIEW

**Comment by kanafghan** [comment]
@toin0u I'm done working on this. If you can accept it, you may merge it.

File: doc/mockups.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+targets that muscle group. From the **Workout Selection** view the user can go to the [Home](#home) view.
+The following mockup illustrates how the **Workout Selection** view can look.
+
+![Sign Up](fiziq_mockup_WorkoutSelection.png)
+
+
+## Log Workout
+The **Log Workout** view is used by the user to enter the *weight* and repetitions (*reps.*) for each set
+of the workout he/she has selected in the [Workout Selection](#workout-selection) view. Initially, this
+view must contain the possibilty for etering weight and reps for one set. There must be a button,
```
Comment (toin0u) [inline doc/mockups.md]: typo `etering` to `entering`

File: doc/mockups.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+## Log Workout
+The **Log Workout** view is used by the user to enter the *weight* and repetitions (*reps.*) for each set
+of the workout he/she has selected in the [Workout Selection](#workout-selection) view. Initially, this
+view must contain the possibilty for etering weight and reps for one set. There must be a button, 
+`Add Set`, that adds a set to the workout. If there exists a record for the workout that is being logged,
+the weights and reps of that record must be presented together with each new corresponding set.
+
+There must be also a `Next Workout` button that takens the user to the
```
Comment (toin0u) [inline doc/mockups.md]: typo `takens` to `takes`

**Comment by toin0u** [comment]
Looks good! :dancer:

**Comment by toin0u** [comment]
:+1:

**Pull request merged**